src/main/g8/default.properties

```diff
@@ -1,3 +1,5 @@
 name = quickstart
 organization = com.example
 package = $organization$.$name;format="norm,word"$
+
+scala_version = 3.3.3
```

Comment:
This reverts https://github.com/http4s/http4s.g8/pull/420 made by @froth  in order to avoid having to change the version in 2 places. I guess is better to remove it, right?

Reply:
That is exactly the heuristic I haven't been able to determine, if Scala Steward will update Mill from SBT update dependencies. I will try to configure and run Scala Steward in a repo with some outdated deps,

Reply:
> I've never used Mill, but we've solved the problem of duplicate constants in templates at work with a technique similar to [this FAQ](https://github.com/scala-steward-org/scala-steward/blob/main/docs/faq.md#can-scala-steward-update-dependencies-in-giter8-templates-). I don't know whether Scala Steward would be able to find a 3.3.3 in a template across SBT _and_ Mill.
> 
> I'm good with whatever @froth thinks here. It would be nice if there's one source of truth, but it's also not the worst duplication in the world.

Can live with both :)

Reply:
Yes, I agree is better to remove it from `default.properties`.